src/components/Button/Button.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+export default function Button({type,disabled,children,clickHandler,className}) {
+    return (
+
+        <button
+            type={type}
+            disabled={disabled || false}
+            onClick={clickHandler}
+            className={className}
+        >
+            {children}
+        </button>
+
+    );
+}
```

Comment:
Dit is een soort doorgeefluik. Je had net zo goed direct het button element kunnen gebruiken